openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -520,11 +545,21 @@ const DatabaseSchemaPage: FunctionComponent = () => {
   }, [viewDatabaseSchemaPermission]);
 
   useEffect(() => {
+    const cursorState = pagingCursor;
```

Comment:
We can avoid this unnecessary assignment to new variable

Reply:
done